update: 2.2.0 changes (#431)
JetBrains/kotlin-multiplatform-dev-docs: topics/development/multiplatform-compatibility-guide.md

* In case deprecated properties are set by some third-party Gradle plugin used in your build, ask the plugin maintainers
  not to set these properties.

As the default behavior of the Kotlin toolchain doesn't include such properties since Kotlin 1.6.20, we don't expect

> any serious impact. Most possible consequences will be visible immediately after the project rebuilds.

Reviewer: ```suggestion
any serious impact. Most consequences will be visible immediately after a project is rebuilt.
```

I'd say only actual consequences will be visible :)

If you're a library author and want to be extra safe, check that consumers can work with your library.

**When do the changes take effect?**

Here's the planned deprecation cycle:

> * 1.8.20: report a warning when Gradle properties are used

Reviewer: ```suggestion
* 1.8.20: report a warning when the deprecated Gradle properties are used
```

Without "these" the phrase becomes ambiguous. Maybe like this?
